include/mbedtls/ssl.h

```diff
@@ -2074,7 +2074,8 @@ void mbedtls_ssl_conf_authmode(mbedtls_ssl_config *conf, int authmode);
  *  namely mbedtls_ssl_handshake(), mbedtls_ssl_handshake_step(),
  *  mbedtls_ssl_read() or mbedtls_ssl_write() may return with the error code
  *  MBEDTLS_ERR_SSL_RECEIVED_EARLY_DATA indicating that some early data have
- *  been received. To read the early data, call mbedtls_ssl_read_early_data().
+ *  been received. To read the early data, call mbedtls_ssl_read_early_data()
+ *  before calling the original function again.
  *
  * \warning This interface is experimental and may change without notice.
```

Comment:
Btw is that still the case? Are we planning to make this API stable before the release and if so are we tracking it?

Reply:
Not completely sure yet, we have #8772 to track it.